source/_integrations/airos.markdown

```diff
@@ -14,7 +14,7 @@ ha_platforms:
   - diagnostics
   - sensor
 ha_integration_type: integration
-ha_quality_scale: bronze
+ha_quality_scale: silver
```

Comment:
Core PR forgot to udpate docs. See: https://github.com/home-assistant/core/pull/153675

Reply:
Oops, thanks for spotting